### PR TITLE
Feature/core 878 handle loading on twin creation

### PR DESCRIPTION
**Pull request by karladler**
draft bc of logs etc. ...

### CODE REVIEW

File: core/digital-twin-lib/src/interfaces/DigitalTwinTemplate.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { DBCPDescriptionInterface } from '../DAppContract';
+import { ContainerPlugin } from '@evan.network/api-blockchain-core';
+
+export interface DigitalTwinTemplateInterface {
```
Comment (Tschuck) [inline core/digital-twin-lib/src/interfaces/DigitalTwinTemplate.ts]: You could also use the `DigitalTwinTemplate` interface from blockchain-core. https://github.com/evannetwork/api-blockchain-core/blob/develop/src/contracts/digital-twin/digital-twin.ts#L55

File: core/digital-twin-lib/src/interfaces/DigitalTwinInterface.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export interface DigitalTwin {
+export interface DigitalTwinInterface {
```
Comment (Tschuck) [inline core/digital-twin-lib/src/interfaces/DigitalTwinInterface.ts]: You could also use the `DigitalTwin` class for this. https://github.com/evannetwork/api-blockchain-core/blob/develop/src/contracts/digital-twin/digital-twin.ts#L133

File: core/digital-twin-lib/src/interfaces/DigitalTwinTemplate.ts (hunk 1 above)
Comment (karladler) [reply core/digital-twin-lib/src/interfaces/DigitalTwinTemplate.ts]: ah nice I was looking for this, but I didn't found it. Probably dev wasn't up to date.

Comment (Tschuck) [reply core/digital-twin-lib/src/interfaces/DigitalTwinTemplate.ts]: Ahh yes, was merged only a hour ago...^^

**Pull request merged**